lib/Dialect/EmitC/Conversion/TosaToEmitC.cpp

```diff
@@ -108,11 +108,149 @@ class RsqrtOpConversion : public OpConversionPattern<tosa::RsqrtOp> {
   }
 };
 
+template <typename SrcOp>
+SmallVector<Value, 2>
+create_broadcast_op_if_needed(SrcOp &srcOp, ArrayRef<Value> operands,
```

Comment:
Do we need to call this in every conversion by hand?

Reply:
I think this is just shared code. I assume we will not only have this in the broadcastable call op conversion.

Reply:
This is in fact shared code. It is used in the `MulOpConversion` and `CallOpBroadcastableConversion`.

Reply:
So we might revise the naming and check that is placed in the anonymous namespace (no need for a free function here), but in principle that's fine :)